src/ontology/uberon-edit.obo

```diff
@@ -112210,9 +112216,12 @@ is_a: UBERON:0007716 ! lumbar subsegment of spinal cord
 
 [Term]
 id: UBERON:0006452
-name: fourth thoracic spinal cord segment
+name: T4 segment of thoracic spinal cord
+def: "The segment of the spinal cord that corresponds to the fourth thoracic vertebra in most mammals." [PMID:19876658]
 subset: defined_by_ordinal_series
+synonym: "fourth thoracic spinal cord segment" EXACT [BIRNLEX:1378]
 synonym: "T4 segment" EXACT [FMA:71170]
+synonym: "T4 segment of spinal cord" EXACT []
```

Comment:
I have deleted this synonym, and the commit is in the branch but not in the PR was some reason.
![image](https://github.com/obophenotype/uberon/assets/94959119/6bcf99c2-1905-4524-b4ba-e336082137bd)